src/averageAges.js

```diff
@@ -20,6 +20,16 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const masMan = century
+    ? people.filter(
+      (person) =>
+        Math.ceil(person.died / 100) === century && person.sex === 'm'
+    )
+    : people.filter((person) => person.sex === 'm');
```

Comment:
1) we can simplify 
2) its better to add a meaningful name to a variables. My example: `menByCentury`
3) apply those recommendation to both `calculateAverageAgeDiff` and `calculateWomenAverageAge` functions
```suggestion
const menByCentury = people.filter((person) =>
  century
    ? Math.ceil(person.died / 100) === century && person.sex === 'm'
    : person.sex === 'm'
);
```